<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadUser</class>
 <widget class="QMainWindow" name="CadUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Usuário</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>271</width>
      <height>186</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Nome Completo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineNome"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Usuário:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineUser"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Senha:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineKey"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSalvar">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
